src/animators/nullAnimator.ts

```diff
@@ -16,8 +16,10 @@ export class Null implements Animator {
     return 0;
   }
 
-  public animate(selection: d3.Selection<any>, attrToAppliedProjector: AttributeToAppliedProjector) {
-    return selection.attr(attrToAppliedProjector);
+  public animate(selection: SimpleSelection<any>, attrToAppliedProjector: AttributeToAppliedProjector): SimpleSelection<any> {
+    // coerce possibly external d3 instance into our own instance of d3 so we can use d3-selection-multi
+    selection = d3.selectAll<d3.BaseType, void>(selection.nodes());
```

Comment:
I *think* so, if consumers misconfigure webpack (which is likely...), we might hold two copies of d3, in which case we'll need to convert